scripts/sync_live_config.py

```
@@ -0,0 +1,255 @@
+def apply_remote(root: Path, diffs: list[Diff], host: str) -> None:
+    to_copy = [d for d in diffs if d.status != "ok"]
+    if not to_copy:
+        return
+    import tarfile
+    import tempfile
+    with tempfile.NamedTemporaryFile(suffix=".tar.gz", delete=False) as tmp:
+        tar_path = tmp.name
+    with tarfile.open(tar_path, "w:gz") as tar:
+        for d in to_copy:
+            tar.add(root / d.repo_rel, arcname=d.repo_rel)
+    remote_tar = "/tmp/sync_live_config_payload.tar.gz"
```

Ruff (0.16.3)

[error] Probable insecure usage of temporary file or directory: "/tmp/sync_live_config_payload.tar.gz"

(S108)